### DISCUSSION TITLE
Make a release, please

### DISCUSSION THREAD

**Discussion by phgrosjean** [rstudio/gradethis#278] [General]
{gradethis}/{learnr} are in intensive development and it is a good news. Many thanks to all contributors. However, for users, it is a long time we are waiting for a release. Also we have to install a development version of {learnr} to use {gradethis}. Latest official release of {learnr} both on CRAN and on GitHub is almost 2 years old now.

Wouldn't it be a good time to plan for a release of both {learnr} and {gradethis}? (preferably no later than June, so that we can plan using it for our courses in September).

**Comment by gadenbuie** [comment]
Thanks for your comment @phgrosjean! I converted it to a discussion since it's a better fit there. Releasing learnr to CRAN is high on our priorities; there are a few more issues we'd like to address before then and you can follow progress on that front here: https://github.com/rstudio/learnr/projects/1

After that, we plan to send gradethis and other grading packages to CRAN shortly thereafter.

**Comment by phgrosjean** [comment]
@gadenbuie I see, thank you. I'll follow this.

**Comment by mbcann01** [comment]
Hi @gadenbuie ,

This discussion is pretty old now, but I'm curious to know if gradthis is still actively being developed and/or if there are plans to send it to CRAN.

Thanks!
